sopel/cli/config.py

```diff
@@ -129,7 +136,14 @@ def handle_init(options):
 
 
 def handle_get(options):
-    """Read the settings to display the value of <section> <key>"""
+    """Read the settings to display the value of <section> <key>
+
+    :param options: parsed arguments
+    :type options: ``argparse.Namespace``
+    :return: Return 0 if everything went fine.
+             Return 1 if the section and/or option does not exist.
+             Return 2 if the settings can't be loaded
```

Comment:
```suggestion
    :return: 0 if everything went fine;
             1 if the section and/or option does not exist;
             2 if the settings can't be loaded
```

This was already inconsistent with its siblings above—the last item had no trailing `.` 😹